draft-ietf-core-dns-over-coap.md

```diff
@@ -132,19 +129,24 @@ Basic Message Exchange
 
 The "application/dns-message" Content-Format    {#sec:content-format}
 --------------------------------------------
-This document defines the Internet media type "application/dns-message" for
-the CoAP Content-Format. This media type is defined as in {{?RFC8484}}
+This document defines a CoAP Content-Format number for the Internet media type "application/dns-message". This media type is defined as in {{?RFC8484}}
 Section 6, i.e., a single DNS message encoded in the DNS on-the-wire format
 {{!RFC1035}}.
 Both DoC client and DoC server MUST be able to parse contents in the "application/dns-message" format.
 
 DNS Queries in CoAP Requests
 ----------------------------
 
-A DoC client encodes a single DNS query in one or more CoAP request messages the CoAP FETCH {{!RFC8132}} method.
+A DoC client encodes a single DNS query in one or more CoAP request
+messages that use the CoAP FETCH {{!RFC8132}} method.
 Requests SHOULD include an Accept option to indicate the type of content that can be parsed in the response.
 
-The CoAP request SHOULD be carried in a Confirmable (CON) message, if the transport used does not provide reliable message exchange.
+The CoAP request SHOULD be carried in a Confirmable (CON) message, if the transport used does not provide reliable message exchange.[^layer-violation]
+
+{: source=" -- cabo"}
+[^layer-violation]: I'm not sure we need this layer violation here; if
+    the client doesn't really need the response, why require it?
+    Also: What are the cases where the "SHoULD" can be ignored?
```

Comment:
Mostly, we wanted to say: "If needed, we want to re-use the retransmission mechanism of CoAP".

Not being interested in the response might not make this much sense in a DNS use case.

Reply:
> (A statement of fact that the reliability of the message layer, e.g., CON, is used to replace the retransmissions of the DNS protocol, would fit well.)

I can do that as a follow-up.